aether-common-module/aether/common/conf/settings.py

```diff
@@ -192,7 +192,7 @@
 CAS_VERSION = 3
 CAS_LOGOUT_COMPLETELY = True
 CAS_SERVER_URL = os.environ.get('CAS_SERVER_URL', '')
-HOSTNAME = os.environ.get('HOSTNAME', '')
+HOSTNAME = os.environ['HOSTNAME']
```

Comment:
You are going to kill me...

Why don't you move that check to here? This is the moment in which you really require it :thinking: (if you are not using UMS).
```python
if DJANGO_STORAGE_BACKEND == 'filesystem':
    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
    HOSTNAME = os.environ['HOSTNAME']
elif ...
```

Reply:
ok, I'll do that

Reply:
I'm just trying to reduce the list of required variables...